src/main/java/org/jenkinsci/account/Myself.java

```diff
@@ -171,5 +177,9 @@ private String fixEmpty(String s) {
         return s;
     }
 
+    private boolean isValidName(String name) {
+        return name.length() < 100;
```

Comment:
probably a safer check? then update the error message to say the name is invalid? 

Or throw the `UserError` from this method instead

```suggestion
        return name.length != null && !name.equals("") && name.length() < 100;
```

Reply:
Yeah, that seems a good one. Then I will also have to remove the redundant fixEmpty calls for first name and last name.